parachain/runtime/kintsugi/src/lib.rs

```diff
@@ -460,9 +460,10 @@ parameter_types! {
     pub const LaunchPeriod: BlockNumber = 7 * DAYS;
     pub const VotingPeriod: BlockNumber = 7 * DAYS;
     pub const FastTrackVotingPeriod: BlockNumber = 3 * HOURS;
-    // TODO: this needs to change before launch. First off, CENTS is for KSM, not for KINT. Second, needs to be sufficciently
-    // high to prevent dos by making 100 proposals
-    pub MinimumDeposit: Balance = 100 * CENTS;
+    // Require 5 vKINT to make a proposal. Given the crowdloan airdrop, this qualifies about 3500
+    // accounts to make a governance proposal. Only 2300 can do two proposals,
+    // and 700 accounts can do ten or more proposals.
+    pub MinimumDeposit: Balance = 5 * UNITS;
```

Comment:
The units of the different currencies is very error prone, I think we should add helper functions like [this](https://github.com/AcalaNetwork/Acala/blob/master/runtime/common/src/lib.rs#L153-L167)

Reply:
Let's drop a ticket in the backlog to do this :+1:

Reply:
Yeah, I agree. Let's do that in a backlog ticket